apps/site/src/lib/components/forms/shared/BaseForm.tsx

```diff
@@ -10,7 +10,12 @@ import hasDeadlinePassed from "@/lib/utils/hasDeadlinePassed";
 const APPLY_PATH = "/api/user/apply";
 const FIELDS_WITH_OTHER = ["pronouns", "ethnicity", "school", "major"];
 
-export default function BaseForm({ children }: PropsWithChildren) {
+export default function BaseForm({
+	applicationType,
+	children,
+}: {
+	applicationType: string;
+} & PropsWithChildren) {
```

Comment:
Suggestion: `PropsWithChildren` is a generic type, so another type/interface can be specified, e.g. `PropsWithChildren<MyProps>`